src/Annotation/Generate.php

```diff
@@ -53,7 +53,7 @@ class Generate
      *
      * @Enum({"public", "protected", "private", "none", true, false})
      */
-    public $get = self::VISIBILITY_PUBLIC;
+    public $get;
```

Comment:
While behavioral this is not a BC break, changing the default value _is_ a BC break. It also makes it really confusing to see what the default value is.

Side note, when we go for a BC break, we should also remove the `true` and `false` values.

Reply:
The behavior hasn't changed. The default visibility is set accordingly when annotations are being processed.